src/components/molecules/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -14,6 +14,27 @@
           {{ item.text }}
         </oc-button>
         <span v-else :aria-current="getAriaCurrent(index)" tabindex="-1" v-text="item.text" />
+        <template v-if="showContextmenu && index == items.length - 1 && items.length > 1">
```

Comment:
```suggestion
        <template v-if="showContextmenu && index === items.length - 1 && items.length > 1">
```
I think the last check (`items.length > 1`) belongs in web, because if we'd introduce actions for the root node we would not be able to do so with this implementation.